index.rst

```diff
@@ -1,61 +1,181 @@
-  those are automatically added.
+.. sectnum::
 
-  Use the following syntax for sections:
+Abstract
+========
 
-  Sections
-  ========
+.. note::
+   **This technote is not yet published.**
 
-  and
+This document describes a program which simulates Rubin's Alert Distribution
+System :cite:`DMTN-093`. This simulator would be useful for development of
+community brokers: it would provide a realistic target for the throughput and
+data formats they would be expected to handle. In addition, it may be a useful
+tool for testing changes to the configuration of Rubin's alert brokers in the
+future.
 
-  Subsections
-  -----------
 
-  and
+High-level goals
+================
 
-  Subsubsections
-  ^^^^^^^^^^^^^^
+A simulator will help groups who are developing community brokers which
+republish Rubin's alert stream. It will provide a realistic benchmark for the
+load they should expect to receive, and will act as a harness for them to run
+tests.
 
-  To add images, add the image file (png, svg or jpeg preferred) to the
-  _static/ directory. The reST syntax for adding the image is
+The simulator should:
 
-  .. figure:: /_static/filename.ext
-     :name: fig-label
+ - Provide a realistic stream of alerts, both in terms of volume and in content.
+ - Be simple to install on realistic broker hardware
+ - Be well-documented
+ - Provide tools to make it clear whether a consumer is keeping up with the
+   stream volume
 
-     Caption text.
+Implementation
+==============
 
-   Run: ``make html`` and ``open _build/html/index.html`` to preview your work.
-   See the README at https://github.com/lsst-sqre/lsst-technote-bootstrap or
-   this repo's README for more info.
+The simulator will be provided as a set of Docker containers and a
+``docker-compose`` file for coordinating those containers. To run the simulator,
+the user spins up the docker containers, and then runs a command which starts
+the flow of data. The data comes from a directory with serialized alert data on
+disk, and is paced into the containerized Kafka broker to match the expected
+rate for Rubin. The data will be repeated indefinitely, with timestamps
+adjusted, so that long-term behavior can be observed, even if only a few minutes
+of source data is available.
 
-   Feel free to delete this instructional comment.
+Alert data will be serialized using the latest Avro alert event schema. The Avro
+schema specification used will be provided along with the containers.
+
+Docker
+------
 
-:tocdepth: 1
+These containers will be provided:
 
-.. Please do not modify tocdepth; will be fixed when a new Sphinx theme is shipped.
+ - A Kafka container, which runs the broker
+ - A Zookeeper container, which the broker connects to for backend state management
+ - A monitoring container, which provides a web-based UI verify that the stream
+   is being fully handled
+ - An injector container, which hosts a command that sends data to the broker
```

Comment:
I'd also suggest a small demo consumer container to demonstrate receiving the alerts, that users could potentially modify for their own code.